nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/DeleteMarkLogic.java

```diff
@@ -69,25 +79,19 @@ public void init(ProcessorInitializationContext context) {
 
     @Override
     public void onTrigger(final ProcessContext context, final ProcessSessionFactory sessionFactory)
-        throws ProcessException {
+            throws ProcessException {
         super.onTrigger(context, sessionFactory);
     }
 
-    /**
-     * Overrides the behavior in the parent class for how each batch of URIs should be processed.
-     *
-     * @param context
-     * @param session
-     * @return
-     */
     @Override
-    protected QueryBatchListener buildQueryBatchListener(final ProcessContext context, final ProcessSession session) {
+    protected QueryBatchListener buildQueryBatchListener(final ProcessContext context, final ProcessSession session, final boolean consistentSnapshot) {
```

Comment:
I think this may have been based off the wrong branch - this produces a compiler error, I think you meant to base this off the DEVEXP-41 work?